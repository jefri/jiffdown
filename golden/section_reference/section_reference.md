- &{#section-ref};

> {section #section-ref}

Section Content