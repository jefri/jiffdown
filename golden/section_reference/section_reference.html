<ul>
<li><p>Section Content</p>
</li>
</ul>
